fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/api/JournalEntriesApiResource.java

```diff
@@ -166,7 +166,7 @@ public String retrieveAll(@Context final UriInfo uriInfo,
             + "journalentries/1?fields=officeName,glAccountId,entryType,amount\n" + "\n" + "journalentries/1?runningBalance=true\n" + "\n"
             + "journalentries/1?transactionDetails=true")
     @ApiResponses({
-            @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = JournalEntryData.class))) })
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = JournalEntriesApiResourceSwagger.GetJournalEntriesTransactionIdResponse.class))) })
```

Comment:
This looks not right to me... This API is not returning the response with paging. I think the one you are looking for is the "retrieveAll" method.

Reply:
When you get the Journal entries using the Transaction Id, the current endpoint return an array (usually two items) with the gl journal entries but It returns with paging 

<img width="1665" alt="Screenshot 2023-02-02 at 12 56 42" src="https://user-images.githubusercontent.com/44206706/216424960-681a0098-ed7a-4a92-815d-d43171bb6f84.png">

Done, I've renamed the class

Reply:
You are still annotating the wrong method. The "retrieveAll" is called not the "retrieveJournalEntryById"

Reply:
Done, updated both annotations